keras_nlp/src/models/backbone.py

```diff
@@ -213,7 +213,16 @@ class like `keras_nlp.models.Backbone.from_preset()`, or from
                 f"`from_preset` directly on `{preset_cls.__name__}` instead."
             )
 
-        backbone = load_serialized_object(preset, CONFIG_FILE)
+        # Forward `config_overrides` and `dtype`.
+        config_overrides = {}
+        if "config_overrides" in kwargs:
```

Comment:
I believe the `kwargs` were supposed to ask as the config override directly (though it look like this broke at some point).

So if you wanted to set bert dropout, for example, you could do
```
model = keras_nlp.models.BertBackbone.from_preset("bert_base_en_uncased", dropout=0.5)
```

Reply:
Yeah, it is a bit weird to see `config_overrides` and I think there is a dangerous default value (`config_overrides={}`) for it in `load_serialized_object`.

I can try to fix it.